.github/workflows/CD-develop.yml

```
@@ -0,0 +1,74 @@
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    env :
+      working-directory: ./
+      APPLICATION: ${{ secrets.APPLICATION_DEV }}
+
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)